src/platform/RTThread/CHIPPlatformConfig.h

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *    Copyright (c) 2020-2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Platform-specific configuration overrides for CHIP on
+ *          RT-Thread platform.
+ */
+
+#pragma once
+
+// ==================== Security Configuration Overrides ====================
+
+#define PENDING_EVENT_FLAG			(1 << 0)
+#define PENDING_EVENT_ISR_FLAG		(1 << 1)
+
+#ifndef CHIP_CONFIG_RTTHREAD_USE_STATIC_TASK
```

Comment:
But some of the places using this are conditioned on CHIP_CONFIG_RTTHREAD_USE_STATIC_THREAD instead, looks like.  Was this compiled in both configurations?